azure-pipelines.yml

```diff
@@ -18,6 +18,12 @@ steps:
 - script: npm ci
   displayName: 'Install dependencies'
 
+- script: npm run lint:fix
+  displayName: 'linting'
```

Comment:
maybe "Lint"? up to you

Reply:
this is how @DaleneyRuben call lint script in `package.json` file. So maybe I have to make my own script on that file?

Reply:
Just to maintain a consistency with the other steps this should start with uppercase **Linting**